webpack.config.js

```diff
@@ -21,7 +21,12 @@ const baseConf = {
     rules: [
       {
         test: /\.jsx?$/,
-        use: ['babel-loader']
+        use: [{
+          loader: 'babel-loader',
+          options: {
+            presets: ['es2015']
```

Comment:
wow... you maybe spotted the error I was having in site-generator (because there is no .babelrc)
I'll test it